gulpfile.js

```diff
@@ -80,3 +83,37 @@ gulp.task('connect', () => {
   });
 });
 
+gulp.task('test', (done) => {
```

Comment:
A few points on this task:

- can you add a comment listing/describing the different known flags  you can pass in the test command?  eg `browserstack`, `https`

- while running the command, it lists out all the packages/files that are being built.  This is a little spammy in the console. Can you see if you can hide that output?  I don't think it's there in prebid.js.

- can you add this task as a dependency step to the `serve` task?